<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="2012-20150403HL" name="com.allinmd.android.SubscribeEditCase" tests="3" failures="1" timestamp="15 Sep 2015 07:09:52 GMT" time="56.508" errors="0">
  <testcase name="addSubscribe" time="8.294" classname="com.allinmd.android.SubscribeEditCase"/>
  <testcase name="checkSubscribe" time="6.973" classname="com.allinmd.android.SubscribeEditCase">
    <failure type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : expected [false] but found [true] in method : .checkSubscribe(SubscribeEditCase:88)
	at com.allinmd.android.SubscribeEditCase.checkSubscribe(SubscribeEditCase.java:88)
]]>
    </failure>
  </testcase> <!-- checkSubscribe -->
  <testcase name="allinLogin" time="41.241" classname="com.allinmd.android.SubscribeEditCase"/>
</testsuite> <!-- com.allinmd.android.SubscribeEditCase -->
